app/controllers/api/admins_controller.rb

```diff
@@ -13,13 +13,13 @@ def create
   end
 
   def index
-    @admin = ::Admin.order_desc.paginate page: params[:page], per_page: Settings.admins.per_page
+    @admin = ::Admin.order_desc
```

Comment:
k còn pagination nữa hả anh?

Reply:
ừ, đang dùng cách truyền hết dữ liệu lên FE, pagination thì để FE xử lí luôn

Reply:
Như vậy cũng OK anh, nhưng nó chỉ OK trong trường hợp đảm bảo là khi dữ liệu DB lớn lên thì số lượng của Admin vẫn k xuất hiện số quá lớn
Chứ nếu có thể phát xinh Admin lớn hơn 100 đi, thì việc này cũng chậm rồi

Reply:
cũng thấy nó có nhược điểm này, 
nhưng mà hiện tại nếu phân trang ở phía BE thì lúc xóa phần tử  sẽ sinh ra nhiều vấn đề khác
nên tạm thời chấp nhận làm theo cách này trước rồi sẽ nghiên cứu sau